lib/Twig/Node/Block.php

```diff
@@ -31,14 +31,14 @@ public function compile(Twig_Compiler $compiler)
     {
         $compiler
             ->addDebugInfo($this)
-            ->write(sprintf("public function block_%s(\$context, array \$blocks = array())\n", $this->getAttribute('name')), "{\n")
+            ->writeln(sprintf('public function block_%s($context, array $blocks = array())', $this->getAttribute('name')), '{')
             ->indent()
         ;
 
         $compiler
             ->subcompile($this->getNode('body'))
             ->outdent()
-            ->write("}\n\n")
+            ->writeln('}')
```

Comment:
missing the second line break, should this be something like done here https://github.com/twigphp/Twig/pull/1748/files#diff-ffa527644d24f261a5759eec9fed03f0R75  ?